src/main/java/graphql/schema/GraphQLTypeUtil.java

```diff
@@ -2,6 +2,10 @@
 
 import graphql.Internal;
 
+import java.util.Stack;
+
+import static graphql.Assert.assertNotNull;
+
 @Internal
```

Comment:
This is internal.  Should it be?  Since we now expect people to twizzle types themselves??

Reply:
I thought it is already public. We should make it public.